TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift

```diff
@@ -187,11 +189,28 @@ public class TwilioVerifyBuilder {
     return self
   }
 
+  /// Set the accessGroup that will be used for the Keychain storage.
+  /// - Parameter accessGroup: Used to specify the access group for the [kSecAttrAccessGroup](https://developer.apple.com/documentation/security/ksecattraccessgroup?language=swift)
+  /// of the KeyChain.
+  /// if `nil` the data will be only available in the app, otherwise using a KeyChain group will enable the option to access data from service extensions.
+  public func setAccessGroup(_ accessGroup: String?) -> Self {
+    self.accessGroup = accessGroup
+    return self
+  }
+
+  /// Enables the synchronization through devices for data stored in the KeyChain.
+  /// - Parameter synchronizable: make use of [kSecAttrSynchronizableAny](https://developer.apple.com/documentation/security/ksecattrsynchronizableany).
+  ///  if `nil` the data won't be synchronized through devices.
+  public func setSynchronizableStorage(_ synchronizable: Bool?) -> Self {
```

Comment:
We should define if this is needed in the next tasks

Reply:
I will add a `TODO`, to revisit both implementations. Thank you.